src/components/organisms/storage/IpfsInfo.tsx

```diff
@@ -36,6 +52,48 @@ const ConnectPeer = ({ onConnected }: { onConnected: () => Promise<void> }) => {
     </form>
   </Box>;
 };
+
+const PinChecker = () => {
+  const [sCid, setSCid] = useState<string>('');
+  const [peerMap, setPeerMap] = useState<any[]>([]);
+
+  const showPinInfo = async () => {
+    const pinResult = await checkPin(sCid);
+    setPeerMap([]);
```

Comment:
peer != pin. Give that state var a correct name ;) and maybe improve the "checkPin" method name to `checkPinStatus(cid)`

Reply:
changed